sdk-setup/src/mb2

```diff
@@ -2672,6 +2674,13 @@ run_build() {
     local hook_exports=()
     local rpms=$(find_rpms "$OPT_OUTPUTDIR" "$OPT_PKGDIR" "$STATEDIR/spec")
     run_hook post-package "${hook_exports[*]}" ${rpms} || fatal "The 'post-package' hook failed"
+
+    if [[ ! $BUILD_NORPMLINT ]]; then
+        CHECK_SUITES="$CHECK_RPMLINT"
```

Comment:
one more issue. Unless I missed something, CHECK_SUITE here is not meant to refer the global variable. The global variable will be set by run_check__process_args_helper, right? In that case I would prefer having `local check_suites=...` here instead.

Reply:
:raised_eyebrow: What was I thinking? Nice catch, thanks.